test/CBLTest.cc

```diff
@@ -88,6 +89,15 @@ CBLTest::~CBLTest() {
     CHECK(CBL_InstanceCount() == 0);
 }
 
+void CBLTest::checkError(CBLError& error, CBLErrorCode expectedCode, CBLErrorDomain expectedDomain) {
+    CHECK(error.domain == expectedDomain);
+    CHECK(error.code == expectedCode);
```

Comment:
I imagine this could probably be done with an == operator on CBLError

Reply:
The operator has been defined in c++/base.hh but `CBLTest.hh` doesn't include the unofficial C++ API headers which are included by `CBLTest_Cpp.hh`. Let make it simple not do change it here.